### PR TITLE
Update: added image and highscore to game over screen

### PR DESCRIPTION
**Pull request by jamie33o**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -574,9 +577,21 @@ scene("game", () => {
...
     add([
-        text("Game over"),
-        pos(10, 10),
+        sprite("game-over"),
+        scale(2.5),
+        pos(30, 0)
     ])
+    add([
+        text("High Score: " + scoreText),
+        pos(gameWidth/2-130, gameHeight/2+90),
```
Comment (rowanam) [inline src/main.js:586]: I would just replace this line with these two lines: 

pos(gameWidth / 2, gameHeight / 2 + 90),
anchor("center"),

- then it'll be centered.

File: src/main.js [hunk 2]
```diff
@@ -574,9 +577,21 @@ scene("game", () => {
...
+    ])
+
+    add([
+        text("Click To Play Again"),
+        pos(gameWidth/2-160, gameHeight/2+170),
```
Comment (rowanam) [inline src/main.js:591]: And same here:

pos(gameWidth / 2, gameHeight / 2 + 170),
anchor("center"),

**Review by rowanam — COMMENTED** [review]
Left 2 notes

File: src/main.js (hunk 1 above)
Comment (Joao4569) [reply src/main.js:586]: Changes made as per Rowan, please test after it is merged into the main branch.

File: src/main.js (hunk 2 above)
Comment (Joao4569) [reply src/main.js:591]: Changes made as per Rowan, please test after it is merged into the main branch.

**Review by Joao4569 — APPROVED** [review]
Approved!

**Pull request merged**